pkg/morph/client/netmap/wrapper/wrapper.go

```diff
@@ -22,9 +22,25 @@ type Wrapper struct {
 	client *netmap.Client
 }
 
+// Option allows to set an optional
+// parameter of ClientWrapper.
```

Comment:
```suggestion
// parameter of Wrapper.
```

Reply:
do you mean to change it in every wrapper?

i copy-pasted that from other contract wrappers

Reply:
Type is named `Wrapper`, so the doc should follow it.

Reply:
ok, rewrited commentary and add commit that fixes the same for other wrappers